.env

```diff
@@ -0,0 +1 @@
+REACT_APP_GOOGLE_REDIRECT = http://localhost:3000/auth/redirect
```

Comment:
env file should be in gitignore, .env.example only should be pushedto git

Reply:
fixed